practice/timezone/timezone/views.py

```diff
@@ -0,0 +1,58 @@
+"""timezone Views."""
+
+from django.http import HttpResponse
+from . import logic
+
+
+def get_server_time(request):
+    """Retrieves the current time in UTC 8601 format."""
+    current_time = logic.get_current_time()
+    return HttpResponse(current_time)
+
+
+def get_timezone_at_lat_lng(request, lat, lng):
+    """Retrieves the timezone name at inputted longitude and latitude coordinates.
+
+    >>> test = get_timezone_at_lat_lng('request', '43.068888', '-121.008911')
+    >>> test.content
+    b'America/Los_Angeles'
+    >>> test = get_timezone_at_lat_lng('request', '1', '1')
+    >>> test.status_code
+    404
+    """
+    try:
+        timezone = logic.get_timezone(float(lat), float(lng))
+        return HttpResponse(timezone)
+    except ValueError as exception:
+        return HttpResponse(str(exception), status=404)
```

Comment:
Advanced comment, but avoid returning an exception in an error message. If it contains a traceback it reveals quite a bit about how you structured your system and can be a security risk.